summingbird-batch-hadoop/src/test/scala/com/twitter/summingbird/batch/HDFSStateLaws.scala

```diff
@@ -16,15 +16,13 @@
 
 package com.twitter.summingbird.batch
 
-import java.util.{ TimeZone, UUID }
-
-import com.twitter.algebird.{ Intersection, Interval }
-import com.twitter.scalding.{ DateParser, RichDate }
+import java.util.{TimeZone, UUID}
```

Comment:
nit: I think it should be with spaces (I think it's where your IDEA style settings going against our scalafmt setup).